numpy/ma/core.py

```diff
@@ -82,6 +82,9 @@
 MaskType = np.bool_
 nomask = MaskType(0)
 
+class MaskedArrayFutureWarning(FutureWarning):
```

Comment:
I briefly considered subclassing this another time (e.g. `MaskedArraySetitemFutureWarning`) for this case as it seems that this module is always going through some level of cleanup and it may be nice to keep this available for other deprecations.